tools/fuzz/fuzz.cpp

```diff
@@ -206,15 +215,36 @@ FuzzStatus ParseFlags(int argc, const char** argv, std::string* in_binary_file,
       } else if (0 == strncmp(cur_arg, "--donors=", sizeof("--donors=") - 1)) {
         const auto split_flag = spvtools::utils::SplitFlagArgs(cur_arg);
         *donors_file = std::string(split_flag.second);
+      } else if (0 == strncmp(cur_arg, "--enable-all-passes",
+                              sizeof("--enable-all-passes") - 1)) {
+        fuzzer_options->enable_all_passes();
       } else if (0 == strncmp(cur_arg, "--force-render-red",
                               sizeof("--force-render-red") - 1)) {
         force_render_red = true;
       } else if (0 == strncmp(cur_arg, "--fuzzer-pass-validation",
                               sizeof("--fuzzer-pass-validation") - 1)) {
         fuzzer_options->enable_fuzzer_pass_validation();
-      } else if (0 == strncmp(cur_arg, "--replay=", sizeof("--replay=") - 1)) {
+      } else if (0 == strncmp(cur_arg, "--repeated-pass-strategy=",
+                              sizeof("--repeated-pass-strategy=") - 1)) {
         const auto split_flag = spvtools::utils::SplitFlagArgs(cur_arg);
         *replay_transformations_file = std::string(split_flag.second);
+      } else if (0 == strncmp(cur_arg, "--replay=", sizeof("--replay=") - 1)) {
```

Comment:
It seems the doc for `--replay` option needs to be adjusted.

Reply:
Oh dear - looks like I put the code for handling repeated pass managers under the condition for when the replay option is provided.

Fixed.